docs/source/reference/kubernetes/multi-kubernetes.rst

```diff
@@ -0,0 +1,146 @@
+.. _multi-kubernetes:
+
+Multi-Kubernetes Clusters
```

Comment:
`Multi-Kubernetes Clusters` is a bit confusing; how about any of these:

* Multiple Kubernetes Clusters
* Multi-cluster Kubernetes
* Kubernetes multi-cluster

"Multi-cluster" seems to be a common term: https://www.cncf.io/blog/2021/04/12/simplifying-multi-clusters-in-kubernetes/, https://www.mirantis.com/cloud-native-concepts/getting-started-with-kubernetes/what-is-kubernetes-multi-cluster/, https://www.kubecost.com/kubernetes-multi-cloud/kubernetes-multi-cluster/

Reply:
I reverted it back to `Multiple Kubernetes Clusters`. The reason for the change was that the `New` badge gets to a new line with this title, but it should be fine, as I feel `Multiple Kubernetes Cluster` has the widest audiance.